### DISCUSSION TITLE
Combine more than two containers in createContainer

### DISCUSSION THREAD

**Discussion by damibazarra** [FormidableLabs/victory#2551] [General]
Hello,

Is it possible to combine more than two containers in createContainer, or do it some other way?

Right now, I'm using createContainer("zoom", "voronoi"), but I would to add the cursor on mouseover. The problem is that if I switch to createContainer("zoom", "cursor") I lose the tooltip I'm using when moving around the graph.

Does anyone have a solution or is it currently not feasible to do this?

Thank you very much. Best regards.

**Comment by damianbc** [comment]
Do you think this will be implemented in a future release?

Thank you very much.

**Comment by leonimurilo** [comment]
I need this exact feature too. It would be great if we could pass more than two items or maybe nest multiple createContainer calls
